### DISCUSSION TITLE
How do you do Find And Replace with Regex?

### DISCUSSION THREAD

**Discussion by hankbinkle** [phcode-dev/phoenix#1039] [Ideas]
Find works as expected but Replace does not seem to interpret special\escape characters like \n (newline).

For example, if I wanted to replace every colon (:) in the following string with new lines, how do you do that?

line 1:line 2:line 3

Thanks!

**Comment by abose** [comment]
@hankbinkle Thanks for the feature request. The feature is currently not supported.
Created https://github.com/phcode-dev/phoenix/issues/1040 to track.
